### ISSUE TITLE
Accessibility Feature Request: Option to increase Font Size

### ISSUE THREAD

**Issue by chrisgrieser** [buchen/portfolio#2286]
**Is your feature request related to a problem? Please describe.**
It's basically an accessibility thing. My eyes are rather bad for my age, and the Retina Display of Macs decreases font size even further. Basically, I have to move my iMac super close to able to read anything within PP.

**Describe the solution you'd like**
Option to increase the Font size app-wide. (Especially the table views, if that#s easier to implement)

**Describe alternatives you've considered**
I already reduced resolution of the iMac ages ago. It solved the issue for most apps, but PP has still an incredibly small font size, compared with other apps. I cannot really think of any other solution to this.

**Comment by buchen** [comment]
Fixed. With the next version, you will have an option to increase the font size.
I am really struggling with the SWT framework here, but with the increased font it look okayisch.

**Comment by chrisgrieser** [comment]
Unfortunately, it seems the new option to change font size does not work properly.

I updated to the newest version of PP, went to the respective setting in the preferences, changed the font size. Then, I restarted as instructed, but the font size did not change. 

Going back to the preferences, the font size has been reset back to "Standard". 😕
Tried it several times with several font sizes, no size change occurs.
